### PR TITLE
[eas-cli] Added alias -m for --message in update/republish 

### PR DESCRIPTION
**Pull request by pusongqi**
# Why

This is to keep the cli consistent by adding -m across all --messages in different scenarios. 

# How

I did this by adding char: m under the message command in update/republish

# Test Plan

The eas cli is tested extensively with unit tests

### CODE REVIEW

**Comment by pusongqi** [comment]
/changelog-entry new-feature Adds -m alias to --message in update/republish

File: packages/eas-cli/README.md
Comment (szdziedzic) [inline packages/eas-cli/README.md:1]: https://github.com/expo/eas-cli/pull/1904#discussion_r1245024114

**Review by szdziedzic — APPROVED** [review]

File: packages/eas-cli/README.md
Comment (pusongqi) [reply packages/eas-cli/README.md:1]: resolved!

**Comment by pusongqi** [comment]
/changelog-entry new-feature Adds -m alias to --message in update/republish and removed README comment

**Pull request merged**